Quick and dirty fix for multifilter provider label

dmfalke opened the pull request:
fixes #380

bobular commented on src/lib/workspace/Variable.tsx:
```diff
@@ -37,7 +39,18 @@ export function VariableDetails(props: Props) {
             Original variable name:
           </div>
-          &nbsp;{variable.providerLabel}&nbsp;
+          &nbsp;
```
It's a minor pain, but I'd pluralise "Original variable name" for the multifilter.

bobular commented on src/lib/workspace/Variable.tsx:
```diff
@@ -63,3 +76,19 @@ export function VariableDetails(props: Props) {
+  variable: MultiFilterVariable,
+  variablesByParentId: Record<string, VariableTreeNode[] | undefined>
+): Variable[] {
+  if (variable.parentId == null) return [];
```
What's this check on `variable.parentId` for?

dmfalke replied:
Oh, that's not needed. It was left over from an incorrect attempt. I will remove it.

bobular commented on src/lib/workspace/Variable.tsx:
```diff
@@ -63,3 +76,19 @@ export function VariableDetails(props: Props) {
+  const variables = variablesByParentId[variable.id] ?? [];
+  for (const child of variables) {
+    for (const grandchild of variablesByParentId[child.id] ?? []) {
+      if (grandchild) variables.push(grandchild);
```
I'm not familiar with the layout of the data structure, so I'll take your word for it that we only want grandchildren (and not the children).

dmfalke replied:
The variable name `grandchild` is not really accurate. What this loop is doing is: for each variable, find its children (those whose `parentId` is the same as the variable's `id`), and add it to the array `variables`. As the array is being iterated, elements that are `push`ed to it will be included in the iteration.

bobular reviewed (changes requested):
I'd just change the plural thing (don't change the help text).

For GEMS1A bacteria in stool - it takes up a lot of screen space.  Maybe add 
```
.EDAWorkspace-ProviderLabel {
  max-height: 5em;
  overflow: scroll;
}
```
or similar?

The other comments are just due to me not understanding!

dmfalke commented:
@bobular I addressed your suggestion about pluralization. I also added some comments and changed some variable names in the help function.

Regarding scrolling the provider names: I'm going to leave as-is for now. If anything, I think the section could be collapsible, but I'm not convinced it's better.
